docs/transaction-verification.md

```diff
@@ -0,0 +1,164 @@
+# Transaction Verification
+This document describes the _transaction policy_ for a Concordium Node.
+That is, how the node decides to reject or accept transactions.
+ 
+## Overview
+Transactions are submitted to a node in the Concordium Network through the GRPC API, from here the transaction is propagated throughout the network via the peers of the node. 
+Transactions can be propagated throughout the network in two ways. 
+ 
+1. The node individually relays a single transactions it receives.
+2. The node relays the transactions as part of a block. This is the case for the `catch-up` mechanism in the Concordium Blockchain.
+ 
+Before transactions are relayed to the network they undergo a verification process within the node. 
+The purpose of the verification process of transactions are the following:
+ 
+1. Prevent congestion of network packets on the network i.e., lowering the network bandwidth.
+This is in combination with the `deduplication` functionality of the node described in the [./network-layer.md](network layer document).
+2. Preventing the consensus layer from performing unnecessary work.
+3. To strengthen the security of the overall network.
+ 
+## Types of transactions
+ 
+The `transactions` of the Concordium Blockchain can be grouped into three categories.
+ 
+1. `CredentialDeployments` When deploying new credentials on the chain. That is, when a user creates a new account.
```

Comment:
```suggestion
1. `CredentialDeployments` When a user creates a new account.
```
Credentials can also be deployed onto existing accounts, which is a normal account transaction that is paid for.